gatsby-node.js

```diff
@@ -32,6 +33,55 @@ exports.sourceNodes = async ({actions, createNodeId, createContentDigest}) => {
   await sourcePrimerRailsData({actions, createNodeId, createContentDigest})
   await sourceOcticonData({actions, createNodeId, createContentDigest})
   await sourceFigmaData({actions, createNodeId, createContentDigest})
+  await sourceDotcomSharedComponentsData({actions, createNodeId, createContentDigest})
+}
+
+async function sourceDotcomSharedComponentsData({actions, createNodeId, createContentDigest}) {
+  // @TODO: eventually get this from a dotcom build artifact
+  const sharedComponents = JSON.parse(fs.readFileSync('shared_components.json'))
```

Comment:
Is this work being planned? I'm worried about this becoming quickly outdated and not trusted.

Reply:
Yes, I'm currently working on it in parallel.